flixel/input/touch/FlxTouch.hx

```diff
@@ -98,6 +106,20 @@ class FlxTouch extends FlxPointer implements IFlxDestroyable implements IFlxInpu
 		setGlobalScreenPositionUnsafe(flashPoint.x, flashPoint.y);
 	}
 
+	/**
+	 * Workaround for browsers treating non-pen touches as 0 pressure.
+	 * 
+	 * If the touch starts off with pressure > 0, it's very likely to be a pen input,
+	 * but it's easier to deal with pressure for regular touches if it's assumed to be 1.0
+	 * as OpenFL claims it should be (possibly a bug?).
```

Comment:
comments are getters and setters are almost never seen by devs, and they aren't included in the api documentation. any documentation on pressure should be put in the var declaration. also this doc seems outdated since your new changes